bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/internal/PersistenceManagerImpl.java

```diff
@@ -296,12 +296,29 @@ private void startEventHandling(PersistenceServiceContainer serviceContainer) {
 
     @Override
     public void allItemsChanged(Collection<String> oldItemNames) {
-        itemRegistry.getItems().forEach(this::added);
+        addPersistenceListenners(oldItemNames);
+        addToPersistenceServiceContainer(oldItemNames);
+    }
+
+    public void addPersistenceListenners(Collection<String> oldItemNames) {
```

Comment:
I'd think the correct spelling would be with just one 'n':

```suggestion
    public void addPersistenceListeners(Collection<String> oldItemNames) {
```